src/repl-frontend/scala/tools/nsc/interpreter/shell/InteractiveReader.scala

```diff
@@ -34,6 +32,8 @@ trait InteractiveReader {
     // hack necessary for OSX jvm suspension because read calls are not restarted after SIGTSTP
     if (scala.util.Properties.isMac) restartSysCalls(readOneLine(prompt), reset())
     else readOneLine(prompt)
+
+  def initCompletion(completion: Completion): Unit = {}
```

Comment:
I would encourage unit saturation here.  To review, `postInit` only ever meant `initCompletion`, but I wonder if `sbt` would like a hook somewhere, there's also `loopPostInit` or similar. Cf https://github.com/scala/scala/pull/5655